storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -94,11 +170,24 @@ Een manier om dat te doen is met de volgende _css snippet_, waarin `hyphens` en
 
 Voor verdere informatie: [Wrapping and breaking text op MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_text/Wrapping_breaking_text)
 
-## Hoe te gebruiken?
+## Het lettertype installeren
+
+Als je de React-componenten gebruikt hoef je niets te doen.
+Deze gebruiken allemaal het juiste lettertype.
```

Comment:
This is currently not exactly true. If you want to use Amsterdam Sans, you have to install it seperately.

Reply:
Yes, nuanced this.